spec/services/hyrax/custom_queries/navigators/find_files_spec.rb

```diff
@@ -0,0 +1,25 @@
+RSpec.describe Hyrax::CustomQueries::Navigators::FindFiles do
+  let(:query_service) { Valkyrie::MetadataAdapter.find(:test_adapter).query_service }
+
+  describe '#find_files' do
+    subject { query_service.custom_queries.find_files(resource: fileset) }
+    context 'when files exist' do
+      let!(:file_metadata1) { FactoryBot.create_using_test_adapter(:hyrax_file_metadata) }
+      let!(:file_metadata2) { FactoryBot.create_using_test_adapter(:hyrax_file_metadata) }
+      let!(:fileset) { FactoryBot.create_using_test_adapter(:hyrax_pcdm_file_set, files: [file_metadata1, file_metadata2]) }
+      it 'returns file metadata resource' do
+        expect(subject).to be_a Array
+        expect(subject.size).to eq 2
```

Comment:
this is redundant with the line below

Reply:
DONE - removed the size check here and in other tests as well